pages/news.vue

```diff
@@ -14,10 +14,10 @@
     <v-row>
       <v-col v-if="loaded">
         <v-card v-for="(newsItem, i) in sortedItems" :key="i" class="mb-6">
-          <v-card-title class="info">
+          <v-card-title class="header">
             {{ newsItem.properties[`title_${$i18n.locale}`] }}
           </v-card-title>
-          <v-card-subtitle class="info">
+          <v-card-subtitle class="header">
```

Comment:
Again, after relooking at the purpose of cards and its colouring, let's simplify and remove the `class`. No need to use `colour=` here since the news is the primary content of this page.